src/screens/Profile/Profile.js

```diff
@@ -191,8 +206,20 @@ class Profile extends React.Component<Props, State> {
 
   handleUserFieldUpdate = (field: Object) => {
     Keyboard.dismiss();
-    const { updateUser, user } = this.props;
-    updateUser(user.walletId, field, () => this.toggleSlideModalOpen(null));
+    const {
+      updateUser,
+      user,
+      navigation,
+      createOneTimePassword,
+    } = this.props;
+    const createOTP = () => {
```

Comment:
As `handleUserFieldUpdate` is a generic function, you will call the `createOneTimePassword` even if the user changes his first name

Reply:
You are totally right, I will move it to `handleUserPhoneFieldUpdate` 👍